docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +11,16 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-0-1]]
+=== Logstash 6.0.1 Release Notes
+* Elasticsearch Output: Ignore event's type field for the purpose of setting document _type if cluster is es 6.x or above
+* Fix bug where plugins that used nested hashes for configuration would not work correctly
+* Ensure logstash always terminates on second sigint, there were some situations where executing two sigints would not kill Logstash
```

Comment:
Change to: Improved shutdown process by ensuring that Logstash terminates on second SIGINT. Prior to this release, there were some situations where executing two SIGNINTs would not kill Logstash.